docs/guides/fedora.md

```
@@ -0,0 +1,267 @@
+
+- You need to keep Fedora up to date. If you are 2 versions behind, your OS is no longer supported by Fedora (updates, security, etc.)
+- E.g. If Fedora 44 is the current stable release, and you are on Fedora 42, your OS is unsupported.
+
+### Installation
+
+1. Download the latest Fedora Workstation (or KDE Plasma Edition) ISO file from the [official Fedora website](https://getfedora.org/en/workstation/download/) and write it to a USB stick.
+
+> [!NOTE]
+> If you want something else than GNOME or KDE as your Desktop Environment, you can check out [Fedora Spins](https://spins.fedoraproject.org/).
+
+2. If you have difficulties starting the live environment from USB, in the Fedora boot menu select: Troubleshooting → Start Fedora in basic graphics mode
```

markdownlint-cli2 (0.23.2)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/2/3

(MD029, ol-prefix)